src/selectors/entities/posts.js

```diff
@@ -391,3 +391,15 @@ export const getMostRecentPostIdInChannel = createSelector(
         return postIdsInChannel[0];
     }
 );
+
+export const getLatestReplyablePost = createSelector(
```

Comment:
Can you rename this to getLatestReplyablePostId so that callers know it's returning an id instead of the whole object?

Reply:
Done